packages/client/src/__tests__/integration/errors/executeRaw-alter-postgres/test.ts

```diff
@@ -33,22 +33,25 @@ test('executeRaw-alter-postgres', async () => {
       Using the example below you can still execute your query with Prisma, but please note that it is vulnerable to SQL injection attacks and requires you to take care of input sanitization.
```

Comment:
This file and test are still called `executeRaw-alter-postgres` maybe we can rename them?